### PR TITLE
Added Lualtek swsi device

### PR DESCRIPTION
**Pull request by LasaleFamine**
#### Summary
Adding new vendor device by Lualtek into the device repository

#### Changes

- added `vendor/lualtek` 
- modified `vendor/index.yml`

### CODE REVIEW

**Comment by LasaleFamine** [comment]
Hello @Jaime-Trinidad is there some sort of ETA that we should wait every time we open a PR? Because I have other devices to add and I want to understand if it's better to open a PR with everything inside (although it would be more difficult to review...). Thanks!

**Review by Jaime-Trinidad — APPROVED** [review]

File: vendor/lualtek/lualtek-swsi-profile.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+# Ping slot data rate index
+#pingSlotDataRateIndex: 0
+# Ping slot frequency (MHz). Set to 0 if the band supports ping slot frequency hopping.
+#pingSlotFrequency: 869.525
+
+# Whether the end device supports class C
+supportsClassC: true
+# If your device supports class C, uncomment the following fields:
+# Maximum delay for the end device to answer a MAC request or confirmed downlink frame (seconds)
+#classCTimeout: 60
```
Comment (Jaime-Trinidad) [inline vendor/lualtek/lualtek-swsi-profile.yaml:41]: if your device is class C you have to uncomment classCTimeout

**Comment by Jaime-Trinidad** [comment]
hey @LasaleFamine thanks for the update, please review the error that your files gives in the class C, and run validation commands again

File: vendor/lualtek/lualtek-swsi-profile.yaml (hunk 1 above)
Comment (LasaleFamine) [reply vendor/lualtek/lualtek-swsi-profile.yaml]: Thanks, done!

**Comment by LasaleFamine** [comment]
@Jaime-Trinidad I had a problem on the validate command that didn't run properly in local. Now it should be ok. Sorry for the inconvenient.

**Comment by LasaleFamine** [comment]
@Jaime-Trinidad sorry for bothering, just want to test if this works so I can open a new PR with other devices. Are there any news? Thanks!

**Comment by Jaime-Trinidad** [comment]
Hi @LasaleFamine there is an error in the validation, can you check the codec file:

`lualtek: lualtek-swsi: EU863-870: payload codec lualtek-swsi-codec invalid
Error: Command failed: bin/runscript -codec-path "./vendor/lualtek/lualtek-swsi.js" -routine encodeDownlink -input '{"data":{"switchValue":0}}'
2023/06/07 09:41:33 Marshal output: json: unsupported value: NaN`

**Comment by LasaleFamine** [comment]
Hi @Jaime-Trinidad it's been a bit frustrating since locally I don't see any error when running the validation. Also seems like the validation script just spawns a lot of processes that sometimes crushes my Mac. I think I will remove that part from the codec directly since I cannot test it with ease.

**Comment by LasaleFamine** [comment]
Hello @Jaime-Trinidad are there any news for this one?

**Review by LDannijs — APPROVED** [review]

**Comment by Jaime-Trinidad** [comment]
Hi @LasaleFamine we replicated the PR and we already merge it, it will be in the next release of TTS, I will close this PR

**Comment by LasaleFamine** [comment]
@Jaime-Trinidad that's awesome! Sorry for not being able to fix it as per CI.

I will try to emulate yours for new devices. Thanks!

**Pull request closed without merging**